wnfs/src/private/directory.rs

```diff
@@ -540,30 +591,35 @@ impl PrivateDirectory {
     /// }
     /// ```
     pub async fn read(
-        self: Rc<Self>,
+        self: &Rc<Self>,
```

Comment:
Huh, what's the reason for `&`ing this? Just consistency?
I think a `mut Rc<PrivateDirectory>` or `&mut Rc<PrivateDirectory>` should deref to `Rc<Self>` just fine, no?

Reply:
Oh yeah right. Not having the `&` would force the caller to `Rc::clone`!